Engines/Wine/QuickScript/GoG Script/script.js

```diff
@@ -111,6 +112,8 @@ module.default = class GogScript extends QuickScript {
             .prefix(this._name, this._wineDistribution, this._wineArchitecture, this._wineVersion)
             .create()
             .wait();
+        
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Trailing spaces not allowed.](https://app.codacy.com/manual/PhoenicisOrg/scripts/pullRequest?prid=4817187)